src/pages/news/index.jsx

```diff
@@ -0,0 +1,161 @@
+import Head from "next/head";
+import React, { useEffect, useState } from "react";
+import Layout from "/src/organisms/layout/index";
+
+// useState
+import useStore from "/src/ions/hooks/state/useStore";
+
+// MUI Imports
+import Stack from "@mui/material/Stack"
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import { CardActionArea } from '@mui/material';
+import Box from "@mui/material/Box"
+
+// Time ago
+import ReactTimeAgo from 'react-time-ago'
+import TimeAgo from 'javascript-time-ago'
+import en from 'javascript-time-ago/locale/en.json'
+
+TimeAgo.addDefaultLocale(en)
+
+
+const Page = () => {
+	const loading  = useStore((state) => state.loading)
+	const error  = useStore((state) => state.error)
+	const setLoading  = useStore((state) => state.setLoading)
+	const [articles, setArticles] = useState(null)
+
+	const axios = require("axios").default;
+
+	const options = {
+		method: 'GET',
+		url: 'https://free-news.p.rapidapi.com/v1/search',
+		params: {q: 'Crypto,Cryptocurrency', lang: 'en'},
+		headers: {
+			'x-rapidapi-host': 'free-news.p.rapidapi.com',
+			'x-rapidapi-key': 'ea63b3156dmsh6da329c497ccafap175621jsn97465587449c'
+		}
```

Comment:
the key should sit invisible in an .env file, or am I wrong?

Reply:
Is ja noch die alte Datei

Reply:
was meinst du?